### PR TITLE
Remove golang grpc for interop test

### PR DESCRIPTION
**Pull request by qiwzhang**
Currently ESP is using both c++ grpc repo and golang grpc repo.   golang grpc repo is just for interop test.
It is using an very old repo.  The most difficult of upgrading bazel is due to this golang grpc.

This PR will switch ESP to use c++ code for interop test. remove golang grpc repo.

With this clean, up it will be much easier to upgrade bazel.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -67,8 +67,9 @@ iap_jwt_verify_nginx_repositories(True)
 
 git_repository(
     name = "com_github_grpc_grpc",
-    commit = "61e77cb7644ef22449fdf7c0ddd7edfe0ae1c9b2",  # v1.21.0
-    remote = "https://github.com/grpc/grpc.git",
+    commit = "edc506849f3429c6b4a6d90ec604aeb1b7b0eb54",  # v1.21.0
+    # TODO: use forked one with some small changes.
+    remote = "https://github.com/qiwzhang/grpc.git",
```
Comment (kh-chang) [inline WORKSPACE]: What's the change? I think it would be helpful if the comment explains why the change was needed.

File: WORKSPACE [hunk 2]
```diff
@@ -133,8 +134,6 @@ bind(
 load(
     "//:repositories.bzl",
     "googletest_repositories",
-    #"grpc_repositories",
```
Comment (kh-chang) [inline WORKSPACE]: nit:del

Comment (qiwzhang) [reply WORKSPACE]: Done

File: WORKSPACE (hunk 1 above)
Comment (qiwzhang) [reply WORKSPACE]: Added grpc pr number.  Tried to submit these changes to grpc upstream
